Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferSetProxy.cpp

```diff
@@ -215,11 +224,34 @@ void RemoteImageBufferSetProxy::didPrepareForDisplay(ImageBufferSetPrepareBuffer
 
         m_pendingFlush->setHandles(WTFMove(handle));
```

Comment:
also the cleanup should happen here, if we had a stale flush we shouldn't cleanup the connection yet. It wil make the flush not signaled.